From c2a8085f3939ad84b86dbf5182d7abed98ec4b71 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Fri, 7 Dec 2018 13:14:58 -0800
Subject: [PATCH] MANTA-4042 confusing message when accessing unauthorized
 paths Reviewed by: Robert Mustacchi <rm@joyent.com> Approved by: Robert
 Mustacchi <rm@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 2a9565d..a0e9332 100644
--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
         "libuuid": "0.1.2",
         "lru-cache": "4.1.5",
         "lstream": "0.0.4",
-        "mahi": "2.2.3",
+        "mahi": "2.2.4",
         "marlin": "git+https://github.com/joyent/manta-marlin.git#master",
         "mime": "1.2.11",
         "moray": "3.4.1",
-- 
2.21.0

